README.md

```diff
@@ -812,9 +812,9 @@ The [Mollweide projection](#geoMollweide) interrupted into two (equal-area) hemi
 
 ### Polyhedral Projections
 
-<a href="#geoPolyhedral" name="geoPolyhedral">#</a> d3.<b>geoPolyhedral</b>(<i>root</i>, <i>face</i>[, <i>angle</i>]) [<>](https://github.com/d3/d3-geo-projection/blob/master/src/polyhedral/index.js "Source")
+<a href="#geoPolyhedral" name="geoPolyhedral">#</a> d3.<b>geoPolyhedral</b>(<i>root</i>, <i>face</i>) [<>](https://github.com/d3/d3-geo-projection/blob/master/src/polyhedral/index.js "Source")
 
-Defines a new polyhedral projection. The *root* is a spanning tree of polygon face nodes; each *node* is assigned a *node*.transform matrix. The *face* function returns the appropriate *node* for a given *lambda* and *phi* in radians. The specified rotation *angle* is applied to the polyhedron; if an *angle* is not specified, it defaults to -π / 6 (for butterflies).
+Defines a new polyhedral projection. The *root* is a spanning tree of polygon face nodes; each *node* is assigned a *node*.transform matrix. The *face* function returns the appropriate *node* for a given *lambda* and *phi* in radians. Use [*projection*.angle](https://github.com/d3/d3-geo/blob/master/README.md#projection_angle) to change the orientation of the map.
```

Comment:
You still want to document that the default projection.angle is -30° here, since that’s the behavior (and it’s different than d3.geoProjection). If you like, you could also document that the behavior may be changing in the future, so it’s a good idea to set projection.angle explicitly to whatever you want.

That said, I personally wouldn’t bother (redundantly) setting the angle to -30° in the various other projections in this repo. If we change the default to 0° in the next major version, that’s when we’d change the projection.angle for Butterfly, Collignon, etc.